manifest.toml

```
@@ -68,6 +68,9 @@ ram.runtime = "50M"
 
     [resources.permissions]
     main.url = "/"
+    admin.url = "/admin/"
+    admin.show_tile = false
+    admin.allowed = "admins"
```

```suggestion
    admin.protected = true
```

(to prevent people from adding `visitors` to it ?)